pkg/volume/empty_dir/empty_dir.go

```diff
@@ -168,26 +206,66 @@ func (ed *emptyDir) setupTmpfs(dir string) error {
 	// By default a tmpfs mount will receive a different SELinux context
 	// from that of the Kubelet root directory which is not readable from
 	// the SELinux context of a docker container.
-	//
-	// getTmpfsMountOptions gets the mount option to set the context of
-	// the tmpfs mount so that it can be read from the SELinux context of
-	// the container.
-	opts := ed.getTmpfsMountOptions()
-	glog.V(3).Infof("pod %v: mounting tmpfs for volume %v with opts %v", ed.podUID, ed.volName, opts)
+	var opts []string
+	if selinuxContext != "" {
```

Comment:
yes

On Mon, Jul 27, 2015 at 12:00 PM, Paul Morie <notifications@github.com>
wrote:

> In pkg/volume/empty_dir/empty_dir.go
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/9384#discussion_r35572994>
> :
>
> > @@ -168,26 +206,66 @@ func (ed *emptyDir) setupTmpfs(dir string) error {
> >  	// By default a tmpfs mount will receive a different SELinux context
> >  	// from that of the Kubelet root directory which is not readable from
> >  	// the SELinux context of a docker container.
> > -	//
> > -	// getTmpfsMountOptions gets the mount option to set the context of
> > -	// the tmpfs mount so that it can be read from the SELinux context of
> > -	// the container.
> > -	opts := ed.getTmpfsMountOptions()
> > -	glog.V(3).Infof("pod %v: mounting tmpfs for volume %v with opts %v", ed.podUID, ed.volName, opts)
> > +	var opts []string
> > +	if selinuxContext != "" {
>
> I agree. I'm actually starting to gather reqs on solving this for other
> volumes, and I think we can have a follow-up to push the SELinux code up to
> a higher level so that the volume plugins won't have to do this work
> individually. Are you cool w/ that @thockin <https://github.com/thockin> ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/9384/files#r35572994>
> .
>